ofrak_patch_maker/Dockerstub

```diff
@@ -25,44 +28,107 @@ fi;
 #M68k GNU 10 Linux
 #Only exists for x86
 RUN if [ "$TARGETARCH" = "amd64" ]; then \
-    apt-get update && apt-get install -y gcc-10-m68k-linux-gnu; \
+    cd /tmp && \
+    apt-get update && apt-get install -y gcc g++ gperf bison flex texinfo help2man make libncurses5-dev python3-dev autoconf automake libtool libtool-bin gawk wget bzip2 xz-utils unzip patch libstdc++6 rsync && \
+    git clone https://github.com/crosstool-ng/crosstool-ng.git && \
+    cd crosstool-ng/ && \
+    git reset --hard 6d1d61cbcacc7ce4622ef024490e0cb56881614b && \
+    ./bootstrap && \
+    ./configure --enable-local && \
+    make && \
+    ./ct-ng m68k-unknown-linux-gnu && \
+    cp .config .config.bak && \
+    sed /^CT_GCC_V_14/d .config.bak | \
+    sed /^CT_GCC_later_than_14/d | \
+    sed /^CT_GCC_14_or_later/d | \
+    sed /^CT_GCC_later_than_13/d | \
+    sed /^CT_GCC_13_or_later/d | \
+    sed /^CT_GCC_later_than_12/d | \
+    sed /^CT_GCC_12_or_later/d | \
+    sed /^CT_GCC_later_than_11/d | \
+    sed /^CT_GCC_11_or_later/d | \
+    sed /^CT_PREFIX_DIR/d | \
+    sed /^CT_GCC_VERSION/d > .config && \
+    echo 'CT_PREFIX_DIR="/opt/rbs/toolchain/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"' >> .config && \
+    echo 'CT_GCC_VERSION="10.5.0"' >> .config && \
+    echo 'CT_EXPERIMENTAL=y' >> .config && \
+    echo 'CT_ALLOW_BUILD_AS_ROOT=y' >> .config && \
+    echo 'CT_ALLOW_BUILD_AS_ROOT_SURE=y' >> .config && \
+    echo 'CT_LINUX_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_BINUTILS_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_GLIBC_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_GLIBC_ENABLE_WERROR=y' >> .config && \
+    echo 'CT_GCC_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_GCC_V_10=y' >> .config && \
+    echo 'CT_GCC_14_or_older=y' >> .config && \
+    echo 'CT_GCC_older_than_14=y' >> .config && \
+    echo 'CT_GCC_13_or_older=y' >> .config && \
+    echo 'CT_GCC_older_than_13=y' >> .config && \
+    echo 'CT_GCC_12_or_older=y' >> .config && \
+    echo 'CT_GCC_older_than_12=y' >> .config && \
+    echo 'CT_GCC_11_or_older=y' >> .config && \
+    echo 'CT_GCC_older_than_11=y' >> .config && \
+    echo 'CT_CC_LANG_OTHERS=""' >> .config && \
+    echo 'CT_GETTEXT_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_GMP_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_ISL_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_LIBICONV_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_MPC_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_MPFR_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_NCURSES_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_ZLIB_PATCH_GLOBAL=y' >> .config && \
+    echo 'CT_ZSTD_PATCH_GLOBAL=y' >> .config && \
```

Comment:
```suggestion
    printf '%s\n' \
      'CT_PREFIX_DIR="/opt/rbs/toolchain/${CT_HOST:+HOST-${CT_HOST}/}${CT_TARGET}"' \
      'CT_GCC_VERSION="10.5.0"' \
      'CT_EXPERIMENTAL=y' \
      'CT_ALLOW_BUILD_AS_ROOT=y' \
      'CT_ALLOW_BUILD_AS_ROOT_SURE=y' \
      'CT_LINUX_PATCH_GLOBAL=y' \
      'CT_BINUTILS_PATCH_GLOBAL=y' \
      'CT_GLIBC_PATCH_GLOBAL=y' \
      'CT_GLIBC_ENABLE_WERROR=y' \
      'CT_GCC_PATCH_GLOBAL=y' \
      'CT_GCC_V_10=y' \
      'CT_GCC_14_or_older=y' \
      'CT_GCC_older_than_14=y' \
      'CT_GCC_13_or_older=y' \
      'CT_GCC_older_than_13=y' \
      'CT_GCC_12_or_older=y' \
      'CT_GCC_older_than_12=y' \
      'CT_GCC_11_or_older=y' \
      'CT_GCC_older_than_11=y' \
      'CT_CC_LANG_OTHERS=""' \
      'CT_GETTEXT_PATCH_GLOBAL=y' \
      'CT_GMP_PATCH_GLOBAL=y' \
      'CT_ISL_PATCH_GLOBAL=y' \
      'CT_LIBICONV_PATCH_GLOBAL=y' \
      'CT_MPC_PATCH_GLOBAL=y' \
      'CT_MPFR_PATCH_GLOBAL=y' \
      'CT_NCURSES_PATCH_GLOBAL=y' \
      'CT_ZLIB_PATCH_GLOBAL=y' \
      'CT_ZSTD_PATCH_GLOBAL=y' \
      >> .config
```

Thoughts on whether this is easier to read? I personally think so.

Note that it is [POSIX standard behavior](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/printf.html#tag_20_94_13) for `printf` to reuse the format string for excess arguments, so this is portable to do in general, even though it might look weird if you're used to `printf()` in C.

> The format operand shall be reused as often as necessary to satisfy the argument operands. Any extra b, c, or s conversion specifiers shall be evaluated as if a null string argument were supplied; other extra conversion specifications shall be evaluated as if a zero argument were supplied. If the format operand contains no conversion specifications and argument operands are present, the results are unspecified.